### ISSUE TITLE
Priority and icon in onError

### ISSUE THREAD

**Issue by Evaske** [mikaelbr/gulp-notify#69]
Hey,

I'm trying to get an icon and priority set in the onError but can't seem to get it to work no matter what I try.

    .on("error", plugins.notify.onError({
            message: "<%= error.message %>",
            title: "Error Compiling",
            priority: 10,
            "icon": path.join(__dirname, 'node_modules/gulp-notify/assets', 'gulp.png')
        }))

This code works if it is a normal notify message but not if it is being done as an error. Am I doing something wrong to allow me to set the priority and icon for an error message?

**Comment by fricks** [comment]
I have a similar problem using gulp-ruby-sass and gulp-notify. The error message don't appear anymore only the success message.

**Comment by mikaelbr** [comment]
@Evaske The notification works, but the icon and priority doesn't? And this happens only onError?

Could you try printing out `options` on this line: https://github.com/mikaelbr/gulp-notify/blob/master/lib/report.js#L32 To see if all options are set correctly.

**Comment by Evaske** [comment]
@mikaelbr Yes the notification works but there is no icon and priority doesn't seem to get set, it just shows as a black notification. 

I will do that print out for you. Bare with me.
